q2_diversity_lib/beta.py

```diff
@@ -0,0 +1,74 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2018-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import biom
+import skbio.diversity
+import sklearn.metrics
+import unifrac
+
+
+from q2_types.feature_table import BIOMV210Format
+from q2_types.tree import NewickFormat
+from ._util import (_disallow_empty_tables,
+                    _safely_constrain_n_jobs)
+
+
+# --------------------Non-Phylogenetic-----------------------
+@_disallow_empty_tables
+@_safely_constrain_n_jobs
+def bray_curtis(table: biom.Table, n_jobs: int = 1) -> skbio.DistanceMatrix:
```

Comment:
Does `skbio.diversity.beta_diversity` have support for `n_jobs`? I don't see anything in the API docs - is it something that scipy handles?

Reply:
`n_jobs` is passed as a "method-specific parameter", and handed off to this guy: [`sklearn.metrics.pairwise_distances`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances.html)